dissemin/settings/common.py

```diff
@@ -181,7 +183,6 @@
                     "django.template.context_processors.tz",
                     "django.template.context_processors.request"
                 ),
-                'debug': True
```

Comment:
Why disable template debug? @wetneb I thought it was OK to let it in production or development mode?